dist/info.css

```diff
@@ -1,18 +1,30 @@
 .support,
-.process,
-h1 {
-  width: 49vw;
+.process {
+  max-width: 844px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.stack_info {
+  max-width: 844px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
 }
-.image {
-  padding: 0 20vw;
+.stack_1 {
+  max-width: 900pxpx;
+  background-color: #f2f1e1;
```

Comment:
u should apply this properties (bag-color, border-radius) for `.stack_info class`
look at result in responsive mode (for example set 850px of ur vieport)

Reply:
got it